compiler/rustc_feature/src/unstable.rs

```diff
@@ -207,9 +216,6 @@ declare_features! (
     (internal, test_2018_feature, "1.31.0", None, Some(Edition::Edition2018)),
     /// Added for testing unstable lints; perma-unstable.
     (internal, test_unstable_lint, "1.60.0", None, None),
-    /// Allows non-`unsafe` —and thus, unsound— access to `Pin` constructions.
-    /// Marked `internal` since perma-unstable and unsound.
-    (internal, unsafe_pin_internals, "1.60.0", None, None),
```

Comment:
This is a libs feature that was added here solely to make it "incomplete" (https://github.com/rust-lang/rust/commit/c93968aee896ff2e2ccba1b195e778eb32d6d6e3), and then later "internal". That's not needed any more.